src/utils/colors.test.js

```diff
@@ -0,0 +1,16 @@
+import { COLORS, getRandomColor } from './colors';
+
+describe('Colors Utils', () => {
+  describe('getRandomColor', () => {
+    it('returns truthy (keyCode)', () => {
+      const firstRandomColor = getRandomColor();
+      const secondRandomColor = getRandomColor();
+
+      if (COLORS.length === 1) {
+        expect(firstRandomColor).toBe(secondRandomColor);
+      } else {
+        expect(firstRandomColor).not.toBe(secondRandomColor);
+      }
+    });
```

Comment:
Testing random methods can be tricky but I believe there is a better way to test it.
Another thing, tests should not have any conditions like if statement. If a method behaves differently because of some param, for example, you should add a `describe` for it, for example:

```js
describe('isWeatherHot', () => {
  describe('When the temperature 30˚C or more', () => {
    it('returns truthy', () => {
      ...
    });
  });

  describe('When the temperature is below 30˚C', () => {
    it('returns falsy', () => {
      ...
    });
  });
})
```

Going back to the test, I think I would rewrite like this:

```js
describe('getRandomColor', () => {
  const { random } = Math;

  beforeAll(() => {
    Math.random = () => 0.5;
  });

  afterAll(() => {
    Math.random = random;
  });

  it('returns "#388E3C" on the first time', () => {
    expect(getRandomColor()).toBe('#388E3C');
  });

  it('returns "#D84315" on the second time', () => {
    expect(getRandomColor()).toBe('#D84315');
  });
});
```

`beforeAll` - setup the test: mocking `random` method from `Math` to always return `0.5`.
`afterAll` - remove the setup: setting `Math.random` back to avoid side effects in other tests.

Reply:
Tests must be always previsible, so random values are no welcome.
A good way, in this case, is to `stub` `Math.random` as @nolleto suggested, but prefer to use `sinon` (https://sinonjs.org/releases/latest/stubs/).

```js
  beforeEach(() => {
    sinon.stub(Math, 'random').returns(0.5)
  });

  afterEach(() => {
    Math.random.restore
  });
```

Reply:
You can also use:
```js
beforeEach(() => {
  jest.spyOn(Math, 'random').mockReturnValue(0.5);
});

afterEach(() => {
  Math.random.mockRestore();
});
```

Reply:
I've updatep PR, check it please )